Editor/Window/ConnectToFleetInput.cs

```diff
@@ -142,9 +144,20 @@ private async Task UpdateFleetMenu()
                 {
                     _fleetAttributes = new List<FleetAttributes>(); 
                 }
+
                 _fleetNameDropdownContainer.choices = _fleetAttributes.Select(fleet => fleet.Name).ToList();
                 _fleetNameDropdownContainer.value = _stateManager.AnywhereFleetName;
                 _fleetIdText.text = _stateManager.AnywhereFleetId;
+
+                var fleet = _fleetAttributes.FirstOrDefault(fleet => fleet.Name == _stateManager.AnywhereFleetName);
+                if (fleet != null)
+                {
+                    var textProvider = new TextProvider();
+                    _statusIndicator.Set(State.Failed,
```

Comment:
This seems incorrect too. Shouldn't it be `State.Success` when the fleet goes `ACTIVE`. And `State.FAILED` when it goes into `ERROR`?

Reply:
Oops! fixed now